docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:20
+WORKDIR /usr/src/app
```

Comment:
/usr/share or /usr/src?

Reply:
I wonder if we should do a node install beforehand so it is not a moving target.

Reply:
I have updated to `/usr/share/app`, the compose file has the `npm install` and `npm start peter`, 

- We use the clean node base image.
- Copy the project contents.
- Build the image. We do this because the `npm install` has the node available from base image.
- Start the image and run `npm install` and `npm start`.

Reply:
Thats the recommendation also peter, because we have the package.lock file. We can only have the node_modules pre baked once we have the official automation app image.